app/assets/stylesheets/views/_homepage.scss

```diff
@@ -251,3 +251,14 @@
     border-top: 4px solid $govuk-brand-colour;
   }
 }
+
+// Temporary overrides for global bar styling on homepage during election.
```

Comment:
Nitpick: this comment feels a bit specific, the changes apply for any time the global bar would appear on the homepage, I suspect we'd leave them in place afterwards in case we need the global bar again at some point.

Reply:
Good point, I've removed the homepage mention.